yevhenii.dolmatov/hash.rb

```diff
@@ -0,0 +1,21 @@
+hash = {
+  'Yes' => 23,
+   'b' => 'travel',
+   'yesterday' => 34,
+   5 => '234',
+   yesss: :fg,
+   try: 30,
+   key: 'some value',
+   ' yesterday1 ' => 34,
+   ' yesteryear ' => 2014 
+  }
+
+  counter = 0
+  arr_of_hash_keys = hash.keys.to_a.map! {|el| el.to_s.downcase}
```

Comment:
```suggestion
  arr_of_hash_keys = hash.keys.to_a.map! { |el| el.to_s.downcase }
```

Better having space inside {} block